src/main/java/io/github/moulberry/notenoughupdates/overlays/MiningOverlay.java

```diff
@@ -279,48 +286,43 @@ public void update() {
             for (NetworkPlayerInfo info : players) {
                 String name = Minecraft.getMinecraft().ingameGUI.getTabList().getPlayerName(info);
                 if (name.contains("Mithril Powder:")) {
-                    mithrilPowder = DARK_AQUA + Utils.trimIgnoreColour(name).replaceAll("\u00a7[f|F|r]", "");
-                    continue;
-                } else if (mithrilPowder == null) {
-                    mithrilPowder = RED + "[NEU] Failed to get data from your tablist";
+                    mithrilPowder = DARK_AQUA + Utils.trimWhitespaceAndFormatCodes(name).replaceAll("\u00a7[f|F|r]", "");
```

Comment:
With your new `trimWhiteSpaceAndFormatCodes` function the replace isnt neccessary anymore (+ is also construction of a regex instead of caching it ig, even tho thats not a huge performance loss either way)